ee/clickhouse/sql/person.py

```diff
@@ -273,6 +273,7 @@
 HAVING argMax(is_deleted, version) = 0
 """
 
+# TODO are these templated correctly now?
```

Comment:
Unrelated code?

Reply:
Yep, pointers for myself to understand where it might need checking. Still very draft :)